test/e2e/egress_test.go

```diff
@@ -664,6 +666,89 @@ func testEgressMigration(t *testing.T, data *TestData, triggerFunc, revertFunc f
 	checkIPNeighbor(fromNode)
 }
 
+func testEgressUpdateBandwidth(t *testing.T, data *TestData) {
+	skipIfEgressShapingDisabled(t)
+	skipIfNotIPv4Cluster(t)
+	skipIfHasWindowsNodes(t)
+	var (
+		bandwidth1 = &v1beta1.Bandwidth{
+			Rate:  "100M",
+			Burst: "200M",
+		}
+		bandwidth2 = &v1beta1.Bandwidth{
+			Rate:  "1G",
+			Burst: "2G",
+		}
+		transMap = map[string]int{
+			"100M": 100,
+			"200M": 200,
+			"1G":   1000,
+			"2G":   2000,
+		}
+	)
+
+	egressNode := nodeName(1)
+	egressNodeIP := nodeIP(1)
+
+	// Create another netns to fake an external network on the host network Pod.
+	fakeExternalName := "fake-external"
+	fakeExternalCmd := "iperf3 -s"
+	cmd, _ := getCommandInFakeExternalNetwork(fakeExternalCmd, 24, "1.1.1.1", "1.1.1.254")
+	if err := NewPodBuilder(fakeExternalName, data.testNamespace, toolboxImage).OnNode(egressNode).WithCommand([]string{"sh", "-c", cmd}).InHostNetwork().Privileged().Create(data); err != nil {
```

Comment:
Do you really need a shell to run the iperf command?

Reply:
Yes, or bash.